jsapp/js/components/modalForms/translationTable.es6

```diff
@@ -108,6 +130,11 @@ export class TranslationTable extends React.Component {
     stores.translations.setTranslationTableUnsaved(false);
   }
 
+  toggleRenameLanguageForm(evt) {
+    evt.stopPropagation();
+    this.state.showLanguageForm ? this.setState({showLanguageForm : false}) : this.setState({showLanguageForm : true});
```

Comment:
`this.setState({showLanguageForm : !this.state.showLanguageForm})` shorter and cleaner :)